graspologic/embed/n2v.py

```diff
@@ -13,17 +13,17 @@
 
 
 def node2vec_embed(
-    graph: Union[nx.Graph, nx.DiGraph],
-    num_walks: int = 10,
-    walk_length: int = 80,
-    return_hyperparameter: float = 1.0,
-    inout_hyperparameter: float = 1.0,
-    dimensions: int = 128,
-    window_size: int = 10,
-    workers: int = 8,
-    iterations: int = 1,
-    interpolate_walk_lengths_by_node_degree: bool = True,
-    random_seed: Optional[int] = None,
+        graph: Union[nx.Graph, nx.DiGraph],
```

Comment:
why do you always tab so far in

Reply:
Default pep8-ifyer in intellij.

Reply:
I can't believe they'd do this, just waste all of our precious characters when we _only get 88_.  it's an outrage, Jetbrains.